### DISCUSSION TITLE
People: Manually select area as face on picture

### DISCUSSION THREAD

**Discussion by I3olle** [photoprism/photoprism#2047] [Ideas]
I am pretty happy with they way faces are recognised most of the time. Especially considering that some people change a lot over the years. Sometimes however faces are not recognised at all and I can't tag it manually. I would like to have the possibility to select an area of the picture and manually mark this area as a face which then can be tagged. Otherwise I have quite a lot of pictures where for example I am in a group picture and the only face recognised as an actual face is mine. And therefore the picture only contains a single person according to Photoprism.

Maybe that feature already exists and I just need to enable it somewhere but I couldn't find anything in the documentation/settings/roadmap.

And yes, I would consider a sponsoring membership for that feature.

**Comment by lastzero** [comment]
There is a GitHub Issue for this already :+1:

**Reply by I3olle** [reply]
Awesome. Can you point me towards it so I can subscribe to it?

**Reply by graciousgrey** [reply]
There you go https://github.com/photoprism/photoprism/issues/1548 :)

**Reply by I3olle** [reply]
Thank you!
